src/xo-app/settings/logs/index.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import { FormattedDate } from 'react-intl'
 import { find, map } from 'lodash'
-
```

Comment:
Why this change?

Reply:
I have removed this line without paying attention while viewing this file code.